src/modules/views/Folder/virtualized/FolderViewBody.jsx

```diff
@@ -141,32 +144,40 @@ const FolderViewBody = ({
         currentFolderId={currentFolderId}
         canUpload={canUpload}
       />
-      {showTable && (
-        <Table
-          rows={rows}
-          columns={columns}
-          dragProps={{
-            enabled: canDrag,
-            dragId: 'drag-drive',
-            onDrop: onDrop({
-              client,
-              showAlert,
-              selectAll,
-              registerCancelable,
-              sharedPaths,
-              t
-            })
-          }}
-          fetchMore={fetchMore}
-          selectAll={selectAll}
-          toggleSelectedItem={toggleSelectedItem}
-          isSelectedItem={isSelectedItem}
-          selectedItems={selectedItems}
-          currentFolderId={currentFolderId}
-          withFilePath={withFilePath}
-          actions={actions}
-        />
-      )}
+      {showTable &&
+        (viewType === 'list' ? (
+          <Table
+            rows={rows}
+            columns={columns}
+            dragProps={{
+              enabled: canDrag,
+              dragId: 'drag-drive',
+              onDrop: onDrop({
+                client,
+                showAlert,
+                selectAll,
+                registerCancelable,
+                sharedPaths,
+                t
+              })
+            }}
+            fetchMore={fetchMore}
+            selectAll={selectAll}
+            toggleSelectedItem={toggleSelectedItem}
+            isSelectedItem={isSelectedItem}
+            selectedItems={selectedItems}
+            currentFolderId={currentFolderId}
+            withFilePath={withFilePath}
+            actions={actions}
+          />
+        ) : (
+          <Grid
+            items={rows}
+            withFilePath={withFilePath}
+            actions={actions}
+            fetchMore={fetchMore}
```

Comment:
Does the "select all" in the bar work in the virtualized grid view if the Grid component does not require the `selectedItems` or `SelectAll`?
also the onDrop prop

Reply:
The select box is inside `GridFile` component so we don't need `selectedItems` or `SelectAll` as the props of `Grid` component